.gitlab-ci.yml

```diff
@@ -19,13 +20,29 @@ build-job:
   except:
     - branches
 
+checksum-signing-job:
+  stage: sign
+  extends: .submit-signing-request
+  needs:
+    - build-job
+  variables:
+    PROJECT: splunk-otel-python
+    ARTIFACT: dist/checksums.txt
+    SIGN_TYPE: GPG
+    DOWNLOAD_DIR: dist
+    REPO_NAME: releng
+    REPO_PATH: splunk-otel-python
+  artifacts:
+    paths:
+      - dist/
+
 deploy-job:
   stage: deploy
   script:
     - pip install hatch keyrings.alt
     - hatch --no-interactive publish
   needs:
-    - build-job
+    - checksum-signing-job
```

Comment:
The deploy job is technically not dependent on the checksum signing job so this could be left unchanged.